File: SEGUNDO CURSO/Programación Multimedia/TEMA 1/Miprimertema/app/src/main/java/com/example/miprimertema/MainActivity.java
package com.example.miprimertema;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button button  = findViewById(R.id.button);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Toast.makeText(MainActivity.this, "¡Hola! Me has pulsado muy bien", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: SEGUNDO CURSO/Programación Multimedia/TEMA 3/MidletMensajero/app/src/main/java/com/example/midletmensajero/MainActivity.kt
package com.example.midletmensajero

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<Button>(R.id.salir).setOnClickListener {

            finish()
        }
        findViewById<Button>(R.id.clientlist).setOnClickListener {

            startActivity(Intent(this,ClientesActivity::class.java))

        }
        findViewById<Button>(R.id.mensajes).setOnClickListener {

            startActivity(Intent(this,PlantillaActivity::class.java))

        }
        findViewById<Button>(R.id.enviaMSG).setOnClickListener {

            startActivity(Intent(this,EnviarMSGActivity::class.java))

        }
        findViewById<Button>(R.id.creditos).setOnClickListener {

            startActivity(Intent(this,CreditosActivity::class.java))

        }
        findViewById<Button>(R.id.instrucciones).setOnClickListener {

            startActivity(Intent(this,InstruccionesActivity::class.java))

        }
    }
}